### PR TITLE
[AU4] Fix volume slider styling

### PR DESCRIPTION
**Pull request by kryksyh**
Resolves: #6805 

- [x] I signed [CLA](https://www.audacityteam.org/cla/)
- [x] The title of the pull request describes an issue it addresses
- [x] If changes are extensive, then there is a sequence of easily reviewable commits
- [x] Each commit's message describes its purpose and effects
- [x] There are no behavior changes unnecessary for the stated purpose of the PR

Recommended:
- [x] Each commit compiles and runs on my machine without known undesirable changes of behavior

### CODE REVIEW

**Review by grliszas14 — APPROVED** [review]

File: au4/src/projectscene/qml/Audacity/ProjectScene/trackspanel/audio/VolumeTooltip.qml [hunk 1]
```diff
@@ -34,7 +34,7 @@ StyledPopupView {
     FontMetrics {
         id: fontMetrics
 
-        font: content.font
+        font: ui.theme.defaultFont
```
Comment (Eism) [inline au4/src/projectscene/qml/Audacity/ProjectScene/trackspanel/audio/VolumeTooltip.qml:37]: or should it be the font from StyledTextLabel?

Comment (kryksyh) [reply au4/src/projectscene/qml/Audacity/ProjectScene/trackspanel/audio/VolumeTooltip.qml]: Of course it should, fixed

File: au4/src/projectscene/qml/Audacity/ProjectScene/trackspanel/audio/VolumeSlider.qml [hunk 2]
```diff
@@ -10,34 +10,72 @@ StyledSlider {
...
+            }
         }
-    }
 
-    onMoved: {
-        if (Math.abs(value - snapPoint) < snapRange) {
-            value = snapPoint
+        onPositionChanged: {
+            if (prv.dragActive) {
+                let relativePos = (mouseX - prv.innerMargin) / (width - prv.handleWidth)
```
Comment (Eism) [inline au4/src/projectscene/qml/Audacity/ProjectScene/trackspanel/audio/VolumeSlider.qml:69]: I see that the code is duplicated from mouseArea.onPressed
it's better to move it to a separate function

File: au4/src/projectscene/qml/Audacity/ProjectScene/trackspanel/audio/VolumeSlider.qml [hunk 3]
```diff
@@ -10,34 +10,72 @@ StyledSlider {
...
+                tooltip.hide()
+            }
         }
-    }
 
-    onMoved: {
-        if (Math.abs(value - snapPoint) < snapRange) {
-            value = snapPoint
+        onPositionChanged: {
+            if (prv.dragActive) {
```
Comment (Eism) [inline au4/src/projectscene/qml/Audacity/ProjectScene/trackspanel/audio/VolumeSlider.qml:68]: you can get rid of one tab by 
`if (!prv.dragActive) { return }`

**Review by Eism — APPROVED** [review]

**Pull request merged**